src/interfaces/swig/Features.i

```diff
@@ -98,6 +98,21 @@ PROTOCOLS_DENSELABELS(CRegressionLabels, RegressionLabels, float64_t, "d\0", NPY
 
 %include <shogun/features/DataGenerator.h>
 
+/* FIXME: DenseFeatures are still included/defined since certain classes
```

Comment:
this is a bit annoying that SWIG needs the template instantiation in order to realise base class hierarchies, but we can delete this once the API conversion is completed